lib/transformers/govukDateInput.js

```diff
@@ -69,10 +72,7 @@ module.exports = ({schemaKey, schema, options, data, schemaErrors} = {}) => {
 
     // Include errors
     if (opts.macroOptions.id in schemaErrors) {
-        // The gds govukDateInput prefixes each item with the id. Given the component has the id 'bla'
-        // The item inputs become "bla-bla[day]"
-        // Not sure how to stop that?
-        opts.errorSummaryHREF = `#${schemaKey}-${opts.macroOptions.items[0].name}`;
+        opts.errorSummaryHREF = `#${opts.macroOptions.items[0].name}`;
```

Comment:
This is to address [this](https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/blob/master/lib/transformers/govukDateInput.js#L72) issue